docs/source/using/basic-usage.adoc

```diff
@@ -33,41 +33,7 @@ For more details about using {project} to manage your local OpenShift cluster, s
 [[diagram-minishift-architecture]]
 .: Minishift architecture
 
-[ditaa, "minishift-architecture"]
-----
-
- host machine
- +-------------------------------------------------------------------------------------+
- |                      /-------------> Minishift VM                                   |
- |                      |                                                              |
- |                      |         /--->  o boots from pluggable Live ISO               |
- | $PATH                |         |      o runs on VirtualBox, HyperV, xhyve or KVM    |
- | └── minishift -=-----|         |                                                    |
- |                      |         |   /------------------------------------------\     |
- |                      |         |   |c1AB                                      |     |
- |                      |-=-------|=--|-=->  Docker                              |     |
- |                      :         |   |   /-----------------------------\        |     |
- |                      |         |   |   | c1FF                        |        |     |
- |~/.minishift/         \-=-------|=--|-=-|-=---> OpenShift             |        |     |
- | |                              |   |   |  |  /-------------------\   |        |     |
- | +-- cache                      |   |   |  |  |                   |   |        |     |
- |     |                          |   |   |  |  | master   node     |   |        |     |
- |     +-- iso                    |   |   |  |  | +-----+  +-----+  |   |        |     |
- |     |   |                      |   |   |  |  | | {d} |  | {d} |  |   |        |     |
- |     |   +-- minishift_b2d.iso -/   |   |  |  | |     |  |     |  |   |        |     |
- |     |                              |   |  |  | |     |  |     |  |   |        |     |
- |     +-- oc                         |   |  |  | |     |  |     |  |   |        |     |
- |         |                          |   |  |  | |     |  |     |  |   |        |     |
- |         +-- v3.6.0                 |   |  |  | |     |  |     |  |   |        |     |
- |             |                      |   |  |  | |     |  |     |  |   |        |     |
- |             +-- oc  -=-------------|-=-|-=+  | +-----+  +-----+  |   |        |     |
- |                                    |   |     \-------------------/   |        |     |
- |                                    |   \-----------------------------/        |     |
- |                                    \------------------------------------------/     |
- |                                                                                     |
- +-------+-----------------------------------------------------------------------------+
-
-----
```

Comment:
I don't think this is the only thing you need to remove, we also have lots of dependencies which use to compile that stuff to convert it to an image. We need to remove those also.

Reply:
Ack

Reply:
Done @praveenkumar